backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/scope/model/impl/ElasticsearchScopeModel.java

```diff
@@ -34,16 +33,16 @@
 
 	private final Set<ElasticsearchIndexModel> indexModels;
 	private final Map<String, URLEncodedString> hibernateSearchIndexNamesToIndexReadNames;
```

Comment:
I think that, after your change, this can be turned into a `Set<URLEncodedString` and renamed to `indexReadNames`.

Reply:
Right! We don't need a map for that anymore.